_config.yml

```diff
@@ -7,9 +7,11 @@
 #
 
 
-
 # --- Site Settings ---
 
+# Github url
+github_url: https://github.com/CodingTrain/website
```

Comment:
Instead of defining your own variable for the repository URL, GitHub/Jekyll provides you with that out of the box. You can access it with `site.github.repository_url`. You can also see that used at `_jekyll/layouts/video.html:59`. That way, there's less to do/forget when the repository gets moved or renamed.

Reply:
For an unknown reason, site.github.repository_url doesn't work for me...

I replaced my 'github_url' with it, and the result was the following:
```
//My View Code URL
"/tree/master/Tutorials/P5JS/p5.js/07/7.4_p5js_Mouse_Interaction_with_Objects"

//My Download URL
https://minhaskamal.github.io/DownGit/#/home?url=/tree//Tutorials/P5JS/p5.js/07/7.4_p5js_Mouse_Interaction_with_Objects


// LIVE Download URL
https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/CodingTrain/website/tree/master/Tutorials/P5JS/p5.js/07/7.4_p5js_Mouse_Interaction_with_Objects 
```
I did check if the variable repository url worked appropriately before adding my own, but it didn't... I'll add it and hope it works on your end.

Reply:
That's interesting... 🤔 I'll definately have to investigate that further. Thanks for clarifying! 👍